src/apps/cs2cs.cpp

```diff
@@ -580,8 +580,11 @@ int main(int argc, char **argv) {
                         }
                         proj_unit_list_destroy(units);
                     } else if (arg[1] == 'm') { /* list prime meridians */
+                        (void)fprintf(stderr,
```

Comment:
I wouldn't emit a warning. If we want to emit one, then we should also do it for datum, ellipsoid, units, etc

Reply:
I'm fine with the second sentence "This list is no longer updated, and some values may conflict with other sources"

Reply:
Listing datums was already removed (e9605e41c87e532e7cc182c971e068de015adf2d).
Listing ellipsoids could potentially show a similar message.
Listing units uses `proj_get_units_from_database` so is fine.

Reply:
Feel free to disregard my comments. I'm not that strong about them.

The only thing that is a bit weird is that internally PROJ still creates PROJ pipeline strings with those deprecated features, but one could argue that is an implementation detail. Well, actually not completely true, as we also export that to the sight of the advanced user...)
```
$ bin/projinfo -s "NTF (Paris)" -t "NTF" -o PROJ -q
+proj=pipeline
  +step +proj=axisswap +order=2,1
  +step +proj=unitconvert +xy_in=grad +xy_out=rad
  +step +inv +proj=longlat +ellps=clrk80ign +pm=paris
  +step +proj=unitconvert +xy_in=rad +xy_out=deg
  +step +proj=axisswap +order=2,1
```

Reply:
I think it's fine to have a short list of hard-coded of prime meridians (e.g. where `paris` ↔ 2.33722917) with the caveat that it doesn't change. It's a bit deprecated, but I wasn't planning to establish this as a deprecated feature. But yes, I can tone down the "legacy PROJ.4" wording throughout.

Reply:
The above pipeline could also have `+pm` expressed in decimal degrees, so the PROJ internals could be changed to do that in the future, should we decide to drop support for these hard-coded lists.